apps/dotcom/client/src/tla/providers/TlaRootProviders.tsx

```diff
@@ -75,10 +99,26 @@ function InsideOfContainerContext({ children }: { children: ReactNode }) {
 
 function SignedInProvider({
```

Comment:
Won't we allow locale changes for signeout users?

Reply:
Hah, it's funny b/c right now there might not be a menu for logged out users, especially if we do this: https://github.com/tldraw/tldraw/pull/4699
I think we need to think of where a general App menu would go for logged out users...